src/Terminus.php

```diff
@@ -296,6 +296,7 @@ private function addBuiltInCommandsAndHooks()
     {
         // List of all hooks and commands. Update via 'composer update-class-lists'
         $this->commands = [
+            'Consolidation\\Filter\\Hooks\\FilterHooks',
```

Comment:
what's this do?

(i could try reading documentation but it's easier to just ask you)

Reply:
`--filter` to Terminus commands. It was removed by mistake in a previous PR and all of the commands tests that use that argument are failing now